hack/version_test_ge_xyz.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+IFS=. read -r x y z <<-a
+$1
+a
+IFS=. read -r xx yy zz <<-a
+$2
+a
+printf "version: %s.%s.%s\nminimum: %s.%s.%s\n" "$x" "$y" "$z" "$xx" "$yy" "$zz" >&2
```

Comment:
Do we need to print the minimum version in all cases? I think it is
useful only on failures.